1d-life/src/1d-life.js

```diff
@@ -39,9 +39,15 @@
    * @param {*} x the current x coordinate in question
    */
   function getNewVal(lifeState, x) {
-    // !!! IMPLEMENT ME
+    const current = lifeState[x];
```

Comment:
nicely implemented!  

this was another cool solution, just showing you something different.  Both look readable, you may like this 

    let left = lifeState[x - 1];
    let current = lifeState[x];
    let right = lifeState[x + 1];

    let allOrNothing = (left && current && right) || (!left && !current && !right);
    
    return !allOrNothing? 1: 0;